pyproject.toml

```diff
@@ -28,3 +63,29 @@ addopts =[
     "--ruff",
     "--ruff-format",
 ]
+
+[dependency-groups]
+deploy = [
+    "gevent<24.10.1",
+    "gunicorn[gevent]==22.0.0",
+    "sentry-sdk==1.27.1",
+]
+dev = [
+    "ipdb>=0.13.13",
+    "pre-commit>=4.0.1",
+    "pytest-django>=4.9.0",
+    "pytest>=8.3.4",
+    "pytest-cov>=6.0.0",
+    "factory-boy>=3.3.1",
+    "vcrpy==6.0.1",
+    "pytest-freezegun>=0.4.2",
+    "pytest-mock>=3.14.0",
+    "pytest-subtests>=0.13.1",
+    "tomlkit>=0.13.2",
+    "freezegun==1.2.2",
+    "django-debug-toolbar==4.3.0",
+    "djhtml==3.0.6",
+    "ruff==0.1.15",
+    "pytest-ruff==0.2.1",
+    "pytidylib>=0.3.2",
+]
```

Comment:
This is an application, so can we also set

```
[tool.uv]
package = false
```

somewhere in here. This tells UV to stop us from accidentally pushing it to PyPI.